Gemfile

```diff
@@ -29,6 +29,7 @@ group :development do
   gem "ruby-prof", :platform => :mri
 
   gem "json",      :platform => :ruby_18
+  gem "celluloid-io"
```

Comment:
Doesn't this mean that every test run will use `celluloid-io`? Since most users don't use that project, I'd very much prefer to use regular standard library sockets by default in our tests.